package.json

```diff
@@ -33,13 +33,13 @@
         "delete:reports:unit": "rimraf tests/results/unit/ || true"
     },
     "dependencies": {
-        "@fortawesome/fontawesome-svg-core": "^6.5.2",
-        "@fortawesome/free-brands-svg-icons": "^6.5.2",
-        "@fortawesome/free-regular-svg-icons": "^6.5.2",
-        "@fortawesome/free-solid-svg-icons": "^6.5.2",
+        "@fortawesome/fontawesome-svg-core": "^6.6.0",
+        "@fortawesome/free-brands-svg-icons": "^6.6.0",
+        "@fortawesome/free-regular-svg-icons": "^6.6.0",
+        "@fortawesome/free-solid-svg-icons": "^6.6.0",
         "@fortawesome/vue-fontawesome": "^3.0.8",
         "@geoblocks/cesium-compass": "^0.5.0",
-        "@geoblocks/mapfishprint": "^0.2.14",
+        "@geoblocks/mapfishprint": "^0.2.15",
```

Comment:
I didn't know that there is already version 0.2.15. I was monitoring the github tag here (https://github.com/geoblocks/mapfishprint/tags). But it seems someone have already released a new version in NPM. And it's based on the develop branch. I have already fixed the issue. May I add my fixes to this branch @pakb ?

Reply:
Sure, tests are failing on the print, so maybe your fixes are what is needed